LCM/scripts/InstallModule.py

```diff
@@ -11,6 +11,12 @@
 except:
     import zipfile
 
+scriptFolderPath = os.path.dirname(os.path.realpath(__file__))
+fullPathDSCLogger = os.path.join(scriptFolderPath, 'nxDSCLog.py')
```

Comment:
we are introducing a dependency of the engine (pull client) on a resource. that doesn't sound correct... maybe some refactoring so that the resource and pull client depend on some component in the engine?

Reply:
nxDSCLog.py is not actually a resource. It is just a helper script used by all the resources for logging.